tc/sgx/trusted_worker_manager/enclave/kme/kme_enclave.edl

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2020 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+enclave {
+    from "signup.edl" import *;
+    from "signup_kme.edl" import *;
+    from "work_order_kme.edl" import *;
+    from "enclave.edl" import *;

```

Comment:
signup.edl and enclave.edl is common for all enclave types

Reply:
Included singup.edl as part of enclave.edl